test/performance/benchmarks/dataplane-probe/prod.config

```diff
@@ -107,3 +107,7 @@ metric_info_list: {
   value_key: "be1"
   label: "activator-errors-with-cc-1"
 }
+metric_info_list: {
```

Comment:
Please don't forget updating the prod benchmark after this PR goes in :)

Reply:
yep.